doc/user/sql/create-source/kinesis-source.md

```diff
@@ -38,6 +38,7 @@ Field | Value | Description
 ------|-------|------------
 `access_key` | `text` | _(Required)_ A valid [access key](https://docs.aws.amazon.com/streams/latest/dev/controlling-access.html) to the Kinesis stream.
 `secret_access_key` | `text` | _(Required)_ A valid [secret access key](https://docs.aws.amazon.com/streams/latest/dev/controlling-access.html) to the Kinesis stream.
+`endpoint` | `text` | _(Optional)_ If the stream exists in a custom AWS region, a valid endpoint for the stream must be provided.
```

Comment:
nit: Can you remove `(Optional)`?

Can we make this more declarative? e.g. `If the stream exists in a custom AWS region, the hostname for the Kinesis endpoint.` And a link to the AWS docs to find that piece of info would be A+.

Reply:
So this is actually just for testing! People are creating custom AWS regions to point to local/test versions of Kinesis. I'm not finding an equivalent looking at AWS Region documentation.

Reply:
Ah--can we give them some guidance for where to find the endpoint value we expect here then?

Reply:
It would be something like "the URL of your localstack installation: https://github.com/localstack/localstack#overview". But honestly I wouldn't mind if we just didn't tell users about this option at all. I think it's a really niche use case; I'd almost say internal use only.